labelbox_dev/task.py

```diff
@@ -221,3 +223,77 @@ def errors(self) -> Union[Dict[str, Any], None]:
             errors['failed_data_rows'] = task_result['failedDataRows']
 
         return errors if errors else None
+
+
+class UpdateDataRowsTask(BulkTask):
+    """ Asynchronous task representing bulk data_rows update job
+    """
+
+    task_type = TaskType.UPDATE_DATA_ROWS
+
+    def __init__(self, json):
+        super().__init__(json)
+
+    @lru_cache()
+    def fetch_result(self) -> Dict[str, Any]:
+        if self.status == TaskStatus.IN_PROGRESS.name:
```

Comment:
why are we throwing an exception here? Isn't IN_PROGRESS a normal condition? Or not?

Reply:
Followed the preexisting convention where we throw an error if we try to fetch results/errors from a task. However, we've discussed before that trying to fetch results/errors from an in_progress task should be allowed, and do the polling for results/errors within the method. I will make this change!